DEVELOPING.md

```diff
@@ -1,210 +1,205 @@
-# Builder Services Development Environment
+# Builder Development
 
 ## Overview
 
-This document captures the steps to start and run a Builder environment for development. The builder environment includes the builder services, as well as the depot web site.
+This document outlines the steps to configure and run a Builder environment for development. The Builder dev environment includes the Builder API service, as well as the Builder Web UI.
 
-There are several ways of creating a Builder dev environment - but supporting all operating systems and environments has proven to be untenable. This document includes one officially supported way of creating a Builder dev environment, and links to unsupported ways of creating the dev environment that you may use at your own risk.
+There are potentially multiple ways of creating a Builder dev environment - but supporting various systems and environments has proven to be untenable. This document includes the recommended and supported way of creating a Builder dev environment.
 
-## Pre-Requisites
+## Prerequisites
 
-Note that while these instructions should technically work with any linux environment that habitat supports, we recommend either a VMWare-based VM or bare metal. Other providers (e.g., Docker, VirtualBox) have posed difficulty. For instance, VirtualBox doesn't support forwarding privileged ports, which makes using the web app outside the linux environment challenging.
+* *Linux VM on VMWare*. You can use either a VMWare Fusion 10 based VM running on Mac OS/X, or bare metal. Other providers (e.g., Docker, VirtualBox) have posed difficulty. For instance, VirtualBox doesn't support forwarding privileged ports, which makes using the web app outside the Linux environment challenging.
 
-VMWare with Vagrant (and [the supplied Vagrantfile](https://github.com/habitat-sh/builder/blob/master/Vagrantfile)) works well, but Vagrant is not required if you're comfortable with setting up your own VM and port forwarding. For VMWare Fusion 10, adding the following lines to `/Library/Preferences/VMware Fusion/networking` seems to suffice (where 172.16.174.130 is the IP of the VM):
+* *Ubuntu Desktop 18.04 LTS*. Other distributions should work, but this is the one we will support.
+
+* *Decision on where you want to run the Web UI*. If you are doing active Web UI development, then you will likely want to run the UI on your Host (Mac OS). If so, there are some extra steps and configuration changes that will be needed, and those are called out below. See the [Web UI README](https://github.com/habitat-sh/builder/blob/master/components/builder-web/README.md) for more info.
+
+## Host OS Provisioning
+
+If you plan to run the UI on the host (Mac) OS, you will need to ensure that the ports are forwarded appropriately.
+
+Add the following lines to `/Library/Preferences/VMware Fusion/networking`:
 ```
```

Comment:
To get your VM IP addr, enter a terminal in the vm, and find the address in the output from `ip address`

Reply:
Will add

Reply:
Done - this section is updated with new NAT config info